tests/shell/pipeline/commands/external.rs

```diff
@@ -212,9 +212,9 @@ fn run_glob_if_pass_variable_to_external() {
 }
 
 #[test]
-fn subexpression_does_not_implicitly_capture() {
```

Comment:
This test was added so we support wrapping long (external) commands in parens to break it across multiple lines e.g.

```nushell
(cargo clippy
  --workspace
  --
  -A clippy::result_large_err
)
```

In this usecase we still want to let the external produce its STDOUT/STDERR 

Furthermore redirecting it affects the TTY state which was the reason @devyn changed things in the IR compiler and aded the test.
https://github.com/nushell/nushell/pull/13467
https://discord.com/channels/601130461678272522/614593951969574961/1266503282554179604